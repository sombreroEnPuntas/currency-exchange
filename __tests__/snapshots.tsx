import { render } from '@testing-library/react'
import React from 'react'

// pages
import Index from '../pages/index'

describe.each`
  Page
  ${Index}
`('$Page.displayName', ({ Page }) => {
  test('snapshot', () => {
    const { container } = render(<Page />)

    expect(container).toMatchSnapshot()
  })
})
