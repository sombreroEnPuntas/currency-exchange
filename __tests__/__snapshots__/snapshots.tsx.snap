// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Function Home] snapshot 1`] = `
<div>
  currency-exchange
</div>
`;
